webapp/components/CommentList/CommentList.vue

```diff
@@ -35,6 +37,9 @@ export default {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-22T15:12:44Z" title="Wednesday, January 22nd 2020, 4:12:44 pm +01:00">Jan 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -15,6 +16,7 @@
         @deleteComment="updateCommentList"
         @updateComment="updateCommentList"
         @toggleNewCommentForm="toggleNewCommentForm"
+        class="comment-tag"
```


Move to beginning of list …